config/application.rb

```diff
@@ -31,5 +31,8 @@ class Application < Rails::Application
 
     # Custom error pages
     config.exceptions_app = self.routes
+
+    config.paths["config/initializers"] << AppConfig.initializers_root.to_s if AppConfig.load_external_config?
```

Comment:
I like what you do in MyJobs with:

```ruby
if AppConfig.load_external_config?
  ...
end
```

Reply:
Note, if you go the route of moving Configuration logic to the Configuration object as discussed in a comment above I recently made, then this is moot.